### DISCUSSION TITLE
Security onion 16.04 integration with the hive

### DISCUSSION THREAD

**Discussion by uskwarrior** [Security-Onion-Solutions/securityonion#2827] [Unsupported Versions] [answered]
Hello All,

I am using security onion 16.04 which unlike security onion 2.3 doesn't have "the hive" installed. Since I don't have enough resources, I can't migrate to SO 2.3. However I need to practice incident response features offered by the hive. 

I wished to know if there is a way we could integrate security onion with the hive ? Is so, can someone point me to the steps involved..? 

Many thanks in advance,

**Comment by weslambert** [comment, marked as answer]
Not in the same manner.  You can use Elastalert to send alerts to TheHive, or you could try the following, but we can't provide support for it: https://github.com/weslambert/soctopus.  In the future, for 16.04-related questions, please use: https://docs.securityonion.net/en/16.04/mailing-lists.html?highlight=mailing#security-onion-mailing-list

**Comment by nimmy-shakya** [comment]
How to integrate with Thehive ?

**Reply by dougburks** [reply]
@nimmy-shakya You are responding to an old discussion from 2021. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
